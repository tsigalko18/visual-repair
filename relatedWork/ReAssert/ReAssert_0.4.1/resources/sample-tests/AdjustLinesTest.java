import static org.junit.Assert.assertEquals;

import java.util.Currency;
import java.util.Locale;

import org.junit.Test;

public class AdjustLinesTest {

	//TODO: to prevent name qualification
	Currency c = null;
	Locale l = null;

	@Test
	public void test() {
		assertEquals("expected", "actual");
		assertEquals(
				Currency.getInstance(Locale.US), 
				Currency.getInstance(Locale.CANADA));
		assertEquals("expected", "actual");
		assertEquals(
				"expected", 
				"actual");
		assertEquals("expected", "actual");
	}
	
	@Test	
	public void test2() {
		assertEquals("expected", "actual");
		assertEquals(
				"expected", 
				"actual");
		assertEquals("expected", "actual");
		assertEquals(
				Currency.getInstance(Locale.US), 
				Currency.getInstance(Locale.CANADA));
		assertEquals("expected", "actual");

	}
	
}
