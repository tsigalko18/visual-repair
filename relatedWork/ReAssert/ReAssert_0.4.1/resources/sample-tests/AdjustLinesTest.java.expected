import static org.junit.Assert.assertEquals;

import java.util.Currency;
import java.util.Locale;

import org.junit.Test;

public class AdjustLinesTest {

	//TODO: to prevent name qualification
	Currency c = null;
	Locale l = null;

	@Test
	public void test() {
		assertEquals("actual", "actual");
		{
			Currency currency0 = Currency.getInstance(Locale.US);
			Currency currency1 = Currency.getInstance(Locale.CANADA);
			assertEquals("CAD", currency1.toString());
			assertEquals("CAD", currency1.getCurrencyCode());
			assertEquals("CAD", currency1.getSymbol());
			assertEquals(currency0.getDefaultFractionDigits(), currency1.getDefaultFractionDigits());
		}
		assertEquals("actual", "actual");
		assertEquals("actual", "actual");
		assertEquals("actual", "actual");
	}
	
	@Test	
	public void test2() {
		assertEquals("actual", "actual");
		assertEquals("actual", "actual");
		assertEquals("actual", "actual");
		{
			Currency currency0 = Currency.getInstance(Locale.US);
			Currency currency1 = Currency.getInstance(Locale.CANADA);
			assertEquals("CAD", currency1.toString());
			assertEquals("CAD", currency1.getCurrencyCode());
			assertEquals("CAD", currency1.getSymbol());
			assertEquals(currency0.getDefaultFractionDigits(), currency1.getDefaultFractionDigits());
		}
		assertEquals("actual", "actual");

	}
	
}
